deploy/curiefense-helm/deploy.sh

```diff
@@ -29,11 +29,10 @@ if ! kubectl get namespaces|grep -q curiefense; then
     echo "curiefense namespace created"
 fi
 
-helm upgrade --install --namespace curiefense --reuse-values $HELM_ARGS \
+# shellcheck disable=SC2086
+if helm upgrade --install --namespace curiefense --reuse-values ${HELM_ARGS} \
```

Comment:
```suggestion
if ! helm upgrade --install --namespace curiefense --reuse-values ${HELM_ARGS} \
```

We only want to describe pods/fetch logs if the deployment fails. Unless I'm missing something, of course.

Reply:
nice catch. Applied in the next commit